components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -256,6 +264,38 @@ class AutoPushFeature(
         service.deleteToken()
     }
 
+    /**
+     * Verifies status (active, expired) of the push subscriptions and then notifies observers.
+     */
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
```

Comment:
nit: `VisibleForTesting.PRIVATE` is the default value.

Reply:
I'm game for this, but some folks have a preference for being explicit so I followed that. 🤷‍♂